examples/shors_code.py

```diff
@@ -0,0 +1,118 @@
+""" Shor's code is a stabilizer code for quantum error correction.
+It uses 9 qubits to encode 1 logic qubit and is resistant
+to at least one bit flip and one sign flip or their combination.
+
```

Comment:
I would make this text flow like this: 

- encode with phase flip a state alpha |0> + beta |1>  to alpha |+++>  + beta |---> 
- concatenate it with the bit-flip code 
- a stabilizer code is defined by the stabilizers, which are Z1Z2, Z2Z3, Z4Z5, Z5Z6 ... and X1X2...X6, X4X5...X9 plus the two encoded logical operators (Xlogical = ZZZZZZZZZ, Zlogical = XXXXXXXXX)
- let's introduce an X error 
- one or two of the ZZ syndromes should light up 
- let's introduce a Z error 
- one or both of the XXXX.. syndromes should result in -1 measurement 
- introduce recovery condititonal on syndrome values 
- measurement in stabilizer codes should be a stabilizer itself, to keep the code in the codespace as far as I understand (but I might be wrong here) and not measurement in the Z basis for each qubits...
- we should be able to show that both the phase flip and bit flip errors are corrected by the recovery method

Reply:
I think the text flow you suggested looks great! I will format the steps into this.

Yes we are able to show both phase flip and bit flip errors.

Reply:
After our discussions: let's keep it simple as it is now - no need to introduce the stabilizer formalism related operators / syndromes.